client/services/code_preprocessors/PythonCodePreprocessorServiceSpec.js

```diff
@@ -572,26 +597,29 @@ describe('PythonCodePreprocessorService', function() {
           expectedPerformance: 'linear',
           evaluationFunctionName: 'katamariDamashi'
         });
-        var performanceTests = [performanceTest];
+        var performanceTests = [[performanceTest]];
         var expectedGeneratedCode = [
+          'performance_test_results = []',
+          'one_task_performance_test_results = []',
           '',
-          'def get_test_input(atom, input_size):',
+          'def get_test_input_from_System_extendString_0(atom, input_size):',
```

Comment:
Previously, we want to avoid re-defining if multiple tasks use the same helper function for performance test. That's why we added taskIndex at the end of helper function name. But now we use a set which stores names of all those helper functions. In this way, code redundancy can be prevented.